client/components/RatingsAndReviews/Reviews/ReviewsList.jsx

```diff
@@ -20,27 +16,40 @@ const useStyles = makeStyles((theme) => ({
     overflow: 'auto',
   },
   gridList: {
-    minHeight: 200,
-    maxHeight: 500
+    minHeight: 550,
+    maxHeight: 550
```

Comment:
Seems odd to me to have the minHeight and maxHeight be the same value.  Would it be possible to just state height: 500?  Or something similar?

Reply:
Good point!